src/Jackett.Common/Content/custom.css

```diff
@@ -51,6 +51,10 @@ body {
    width: 60px;
 }
 
+.setup-item-label:empty {
```

Comment:
please update src/Jackett.Common/Content/index.html in order to flush the browser cache to force loading new css
by updating the changed tag. suggestion:  PR12083
https://github.com/Jackett/Jackett/blob/7304ec222b8a3d25552c62ea9d8603e7f8c4a28e/src/Jackett.Common/Content/index.html#L31